### DISCUSSION TITLE
Change mail of git account

### DISCUSSION THREAD

**Discussion by newstable** [community/community#36445] [Repositories] [answered]
I wanna change git mail to another mail but I can't do it.
Where can I change my mail to another mail?
Please let me know if you know about that.
Thank you

**Comment by ystndr** [comment, marked as answer]
I guess [this page](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/changing-your-primary-email-address) contains the information you need

<img width="766" alt="image" src="https://user-images.githubusercontent.com/31244806/196343759-aa4cf19b-9e92-4067-a279-c7afb7d5d2ad.png">

**Reply by newstable** [reply]
Thank you.
I've changed.
